lib/fbe/middleware/sqlite_store.rb

```diff
@@ -68,8 +72,10 @@ def perform(&)
     @db ||=
       SQLite3::Database.new(@path).tap do |d|
         d.transaction do |t|
-          t.execute 'CREATE TABLE IF NOT EXISTS cache(key TEXT UNIQUE NOT NULL, value TEXT);'
+          t.execute 'CREATE TABLE IF NOT EXISTS cache(' \
+                    'key TEXT UNIQUE NOT NULL, value TEXT, touched_at TEXT NOT NULL);'
```

Comment:
@Yegorov the same here: the indentation by 10 spaces looks ugly. Two spaces always, no exceptions. Read this too: https://www.yegor256.com/2014/10/23/paired-brackets-notation.html

Reply:
@yegor256 It's probably better to use heredoc it here too
```ruby
t.execute <<~SQL
  CREATE TABLE IF NOT EXISTS cache(
    key TEXT UNIQUE NOT NULL, value TEXT, touched_at TEXT NOT NULL
  );
SQL
```

or:
```ruby
t.execute(
  'CREATE TABLE IF NOT EXISTS cache(' \
  '  key TEXT UNIQUE NOT NULL, value TEXT, touched_at TEXT NOT NULL' \
  ');'
)
```

Reply:
@Yegorov I agree

Reply:
@yegor256 thanks, fixed!